### PR TITLE
Update documentation for create component

### LINKED ISSUE #3087: Update documentation to reflect new create component view
- `./images/gif/create-component-demo.gif` was recorded before the new create component webview was merged. We should update that `.gif`.
- The README references the git import view, which was removed
- There is probably more documentation that is out of date

### PR DESCRIPTION
**Pull request by datho7561**
Closes #3087

Signed-off-by: David Thompson <davthomp@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,11 +71,16 @@ To install OpenShift Container Platform 4 in the public cloud, in your datacente
...
-### **Import From Git**
+### **Create Component**
 
-Users can directly deploy the git repo code on top of OpenShift/Kubernetes cluster. The guided workflow allows them to provide git repository and the extension detects the files in the repo and recoommends a deployment strategy to deploy on cluster. Users can also provide their own custom deployment strategy and create a component directly from the webview. This provides a One-click deployment from Git to OpenShift
+This guided workflow allows you to configure a project so that it can be run and debugged on OpenShift/Kubernetes.
+You can use an existing local code base,
+an existing code base on a remote git repository,
+or create a new project from a template.
+This workflow generates a file called `devfile.yaml` (a [Devfile](https://devfile.io/)) that contains instructions on how to deploy the project to OpenShift/Kubernetes based on the project's language or framework.
+Once you have this set up, you can debug your project on OpenShift/Kubernetes by right clicking
```
Comment (JessicaJHee) [inline README.md:81]: I think this sentence is missing the second half?

Comment (JessicaJHee) [inline README.md:81]: I think this sentence is missing the second half?

File: package.json [hunk 2]
```diff
@@ -1531,6 +1531,15 @@
 							"altText": "create component from workspace folder"
 						}
 					},
+					{
+						"id": "createComponentFromGit",
+						"title": "Create Component from git repository",
+						"description": "Use the 'Create Component' UI and select 'From Existing Remote Git Repository' in order to import a project from git so that you can deploy it to OpenShift/Kubernetes.\n[Create Component](command:openshift.componentTypesView.registry.openHelmChartsInView)",
```
Comment (JessicaJHee) [inline package.json:1537]: The create component button calls `command:openshift.componentTypesView.registry.openHelmChartsInView`

File: package.json [hunk 3]
```diff
@@ -1531,6 +1531,15 @@
 							"altText": "create component from workspace folder"
 						}
 					},
+					{
```
Comment (JessicaJHee) [inline package.json:1534]: For here, it would be nice to either do one item that explains all 3 workflows or create a separate item for each workflow.

**Review by msivasubramaniaan — APPROVED** [review]
LGTM

**Pull request merged**